src/screens/home/views/OverlayView.tsx

```diff
@@ -277,7 +277,7 @@ export const OverlayView = ({status, notificationWarning, turnNotificationsOn, b
                 <SystemStatusOff i18n={i18n} />
               </Box>
             )}
-            {status === SystemStatus.Unauthorized && (
+            {!userStopped && status === SystemStatus.Unauthorized && (
```

Comment:
What happens when the user has stopped EN?

Reply:
There's an on / off button that take priority + there's an on / off panel in the drawer 

**on / off button** 
<img width="200"  src="https://user-images.githubusercontent.com/62242/100474646-226e5c00-30af-11eb-8708-7aa3c2c9613d.jpeg">

**on panel** 
<img width="200" src="https://user-images.githubusercontent.com/62242/100474573-ea671900-30ae-11eb-81f7-75768d7f8cdb.jpeg">